test/trajectory-subsetter/TrajectorySubsetter_Regression.ipynb

```diff
@@ -247,7 +253,11 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "ffb9742c",
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "raises-exception"
```

Comment:
Again jumping in far too late. Discussed a little bit in Slack.

I think this metadata tag is super cool to allow all cells to run (so an early failure will still let you know whether later tests pass or fail, in order to better understand the scope of failures in the full suite). That said, we still need to make the notebook fail in some way at the end if any tests fail, so things like Bamboo know that there was at least one failed test.